tests/unit/Search/SearchUIUtilsTest.ts

```diff
@@ -641,10 +641,10 @@ const reportsListItems = [
                 shouldShowYear: true,
                 tag: '',
                 to: {
-                    accountID: 18439984,
-                    avatar: 'https://d2k5nsl2zxldvw.cloudfront.net/images/avatars/avatar_3.png',
-                    displayName: 'Admin',
-                    login: adminEmail,
+                    accountID: 0,
+                    avatar: '',
+                    displayName: undefined,
```

Comment:
Are there still tests for reports in the processing state with the manager set? @nkdengineer